packages/medusa/src/api/middlewares/sales-channel/get-requested-sales-channel.ts

```diff
@@ -0,0 +1,24 @@
+import SalesChannelService from "../../../services/sales-channel"
+import { NextFunction, Request, Response } from "express"
+import { ExtendedRequest } from "../../../types/global"
+import { SalesChannel } from "../../../models"
+
+export async function getRequestedSalesChannel(
+  req: Request,
```

Comment:
I will remove that since it is a bit over kill for the sales channels. If it become needed then will see at that moment.